src/Microsoft.SqlTools.ServiceLayer/Admin/Contracts/DatabaseInfoWrapper.cs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.SqlTools.ServiceLayer.Admin.Contracts 
+{
+    public class DatabaseInfoWrapper
+    {
+        public string name;
```

Comment:
A couple of things:
- We don't use public fields, we always use properties instead. So this should become `public string Name { get; set; }` for example
- Normally, we leave the business logic out of contracts classes so they're "pure". I'd suggest having a Utils method that converts DatabaseInfo to this wrapper type
- I'd also have a generic method that works like `GetValueAsString` and handles the TryGetValue and setting of the value so it's easier to assign the fields.